packages/headless/src/app/reducers.ts

```diff
@@ -81,3 +82,4 @@ export const history = undoable({
 export const recommendation = recommendationReducer;
 export const productRecommendations = productRecommendationsReducer;
 export const productListing = productListingReducer;
+export const productListingSort = productListingSortReducer;
```

Comment:
Just a note here, if you are registering this reducer using:

```
engine.addReducers({productListingSort});
```
then the key in state will be `productListingSort` (i.e. engine.state.productListingSort).

Personally, I think the `ProductListing` prefix can be removed because it's implied by the engine, but it's up to you.
It's possible to alias by passing a key and value when adding the reducer:

```
engine.addReducers({sort: productListingSort})
```

However, if there is more than one controller registering the reducer, then you would need to duplicate the alias. This is error prone. The goal of this file was to centralize the aliasing.